server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -171,5 +174,26 @@
             );
             handleResponse(session, response, false);
         });
+
+    /**
+     * Revoke an access token
+     */
+    router.post(
+        "/:tenantId/document/:id/revokeToken",
+        validateRequestParams("tenantId", "id"),
+        getTokenFromRequest(),
+        validateTokenRevocationClaims(),
+        verifyStorageToken(tenantManager, config),
+        throttle(tenantThrottler, winston, tenantThrottleOptions),
```

Comment:
Let's throttle as early as possible

Reply:
I placed it here just because we do it for all other APIs. Do you think we should move throttle for all other APIs?

Reply:
We should do them in this order: cluster throttle, token validate, and cluster throttle. Since we decided not to  enable the cluster throttling. We can keep it as is for now.

We're in the process of enabling cluster throttling for all the public endpoints. Some of them have been enabled while we're working on others.